client/src/utils/helper.ts

```diff
@@ -30,3 +32,51 @@ export const shuffleArray = <T>(array_: T[]) => {
 
   return array;
 };
+
+const WeekDays: { [key: string]: number } = {
+  Sun: 0,
+  Mon: 1,
+  Tue: 2,
+  Wed: 3,
+  Thu: 4,
+  Fri: 5,
+  Sat: 6,
+};
+
+export const transformSlots = (slots: any) => {
```

Comment:
I don't like using any type, because it defeats the purpose of typescript!
Can we define the type here

Reply:
Oh okay sure